src/main.js

```diff
@@ -19,7 +19,7 @@ var buttonMakePoster = document.querySelector('.show-form');
 var buttonNevermind = document.querySelector('.show-main');
 var buttonBackToMain = document.querySelector('.back-to-main');
 var buttonShowPoster = document.querySelector('.make-poster');
-var imageInput = document.getElementById('poster-image-url').value;
```

Comment:
the .value at the end of line 22 was causing a syntaxError in the console

Reply:
Ah good catch, forgot to delete.